go.mod

```diff
@@ -6,9 +6,11 @@ require (
 	github.com/MicahParks/keyfunc/v2 v2.1.0 // indirect
 	github.com/golang-jwt/jwt/v5 v5.2.1 // indirect
 	github.com/gorilla/mux v1.8.1
+	github.com/gorilla/sessions v1.2.2
 	github.com/joho/godotenv v1.5.1
 	github.com/kr/pretty v0.3.1 // indirect
 	github.com/rogpeppe/go-internal v1.12.0 // indirect
 	github.com/stytchauth/stytch-go/v11 v11.5.2
```

Comment:
Not sure exactly how these are generated, but we probably should only have 1 stytch-go dependency

Reply:
Running `go mod tidy` should fix this!